examples/dense_bubble.py

```diff
@@ -112,8 +112,11 @@
     # Set up forcing
     compressible_forcing = CompressibleForcing(state)
 
-    diffusion_dict = {"u": InteriorPenalty(state, state.V[0],direction=[2], params={"kappa":Constant(75.), "mu":Constant(1./delta)}),
-                      "theta": InteriorPenalty(state, state.V[2],direction=[2], params={"kappa":75., "mu":0.005})}
+    V = state.V[0]
+    bcs = [DirichletBC(V, Expression(("0.0","0.0")), "bottom"),
+           DirichletBC(V, Expression(("0.0","0.0")), "top")]
```

Comment:
I think `Expression(("0.0", "0.0"))` is misleading; I'd rather see 0.0 here (you're setting the DoF value, not the field)